web/cobrands/sass/_layout.scss

```diff
@@ -759,7 +895,7 @@ textarea.form-error {
 // to make space for the #report-a-problem-sidebar.
 #side-form, #side {
   .with-notes & {
-    width: 27em;
+    width: 50%;
```

Comment:
This breaks pages such as the /admin/report_edit and the Zurich /report/new , both of which use `with-notes`.